app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -148,7 +149,7 @@ class HomeFragment : Fragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         postponeEnterTransition()
-        StrictMode.allowThreadDiskReads().resetAfter {
+        MainScope().launch(IO) {
```

Comment:
Fragments provide a scope we can use, which should be sufficient here.
```suggestion
        lifecycleScope.launch(IO) {
```
This scope is automatically cancelled when the fragment is destroyed.

Reply:
Ah thanks! Will update it.